test/functional/toolchains/cmake/cmakedeps/test_cmakedeps.py

```diff
@@ -620,6 +620,46 @@ def test_error_missing_build_type(matrix_client):
     assert "matrix/1.0: Hello World Release!" in client.out
 
 
+@pytest.mark.tool("cmake", "3.23")
+def test_imported_configurations(matrix_client):
+    # https://github.com/conan-io/conan/issues/11168
```

Comment:
The comment is pointing to another issue? Shouldn't this be #14606 and #16688?

Reply:
Yes, copy & paste, thanks for pointing this out, I'll fix it in next commit.